Sources/PrimerSDK/Classes/Third Party/PromiseKit/Cancellation/CancellableCatchable.swift

```diff
@@ -378,7 +378,12 @@ internal extension CancellableCatchMixin {
      - Parameter body: The handler to execute if this promise is rejected with the provided error type.
      - SeeAlso: [Cancellation](https://github.com/mxcl/PromiseKit/blob/master/Documentation/CommonPatterns.md#cancellation)
      */
-    func recover<V: CancellableThenable, E: Swift.Error>(only: E.Type, on: Dispatcher = conf.D.map, policy: CatchPolicy = conf.catchPolicy, _ body: @escaping(E) throws -> V) -> CancellablePromise<C.T> where V.U.T == C.T {
```

Comment:
Can we exclude PromiseKit from SwiftLint capture?

Reply:
We can and we will for sure disable linting in those files, something I will do in the following sprint.

Reply:
can we revert the linting changes here in that case? We should try not to modify it all if possible